app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -176,3 +179,8 @@ fun String.parseIntoQualifiedID(): QualifiedID {
     val components = split("@")
     return QualifiedID(components.last(), components.first())
 }
+
+data class ContentParams(
+    val navBackStackEntry: NavBackStackEntry,
+    val arguments: List<Any> = emptyList()
```

Comment:
I was thinking about declaring other variables also directly like:
```
data class ContentParams(
    val navBackStackEntry: NavBackStackEntry,
    val serverConfig: ServerConfig
)
```
This way (using `List<Any>`) is also fine, actually more versatile, it just makes this extra step of checking if the list contains the necessary argument type - I think it's easier for devs if they see which arguments are necessary when creating the instance of `ContentParams` but it may be just my personal preference 😄